shared/structures/src/UitpasClientCredentialsStatus.ts

```diff
@@ -0,0 +1,39 @@
+export enum UitpasClientCredentialsStatus {
+    NOT_CHECKED = 'NOT_CHECKED',
+    NOT_CONFIGURED = 'NOT_CONFIGURED',
+    NO_PERMISSIONS = 'NO_PERMISSIONS',
+    MISSING_PERMISSIONS = 'MISSING_PERMISSIONS',
+    OK = 'OK',
```

Comment:
```suggestion
    NotChecked = 'NotChecked',
    NotConfigured = 'NotConfigured',
    NoPermissions = 'NoPermissions',
    MissingPermissions = 'MissingPermissions',
    Ok = 'Ok',
```
We gebruiken normaal altijd CamelCase voor enums